data/src/main/java/io/gnosis/data/backend/dto/TransactionDto.kt

```diff
@@ -26,10 +27,10 @@ data class MultisigTransactionDto(
     val nonce: BigInteger,
     val isExecuted: Boolean,
     val isSuccessful: Boolean? = null,
-    val executionDate: String? = null,
-    val submissionDate: String? = null,
-    val creationDate: String? = null,
-    val modified: String? = null,
+    val executionDate: Date? = null,
```

Comment:
why java.util.Date and not Instant from threetenbp?

Reply:
1. AFAICT the converter class dosen't support threeten classes
2. This way the timezone handling stuff can be local to the app and doesn't need to be in the data module